source/NVDAObjects/UIA/winConsoleUIA.py

```diff
@@ -254,6 +254,17 @@ class WinConsoleUIA(KeyboardHandlerBasedTypedCharSupport):
 	#: the caret in consoles can take a while to move on Windows 10 1903 and later.
 	_caretMovementTimeoutMultiplier = 1.5
 
+	def _get_windowThreadID(self):
```

Comment:
Not that I think this should happen now, but I wonder whether we want a base class for console windows that the legacy and UIA impls inherit from.